Lib/socket.py

```diff
@@ -714,12 +714,12 @@ def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,
             return sock
 
         except error as _:
```

Comment:
Please don't use `_`, it would look much better with a normal variable name such as `e`.